doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -0,0 +1,320 @@
+.. _ug_nrf54l_cryptography:
+
+nRF54L Series cryptography
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The cryptographic peripherals of the nRF54L Series are supported through a set of standard PSA Crypto APIs, with some additional vendor specific extensions.
+
+The nRF Security library offers a set of :ref:`nrf_security_drivers`.
+On the nRF54L devices, in addition to the nrf_oberon driver covering the software-based cryptography implementations, the CRACEN driver (nrf_cracen) provides entropy and hardware-accelerated cryptography using the CRACEN (Crypto Accelerator Engine) peripheral.
```

Comment:
Should the nRF Security library page link to this page and mention that there is a nRF54L-specific implementation?

Reply:
Link added in PR https://github.com/nrfconnect/sdk-nrf/pull/17804, a draft PR with work in progress.